moirae/models/ecm/advancedSOH.py

```diff
@@ -27,6 +28,31 @@ class ECMASOH(HealthVariable):
     h0: HysteresisParameters = Field(default=HysteresisParameters(base_values=0.0),
                                      description='Hysteresis component')
 
+    def get_theoretical_energy(self,
+                               soc_limits: Tuple[float, float] = (0., 1.),
+                               temperature: Optional[float] = None) -> Union[float, np.ndarray]:
+        """
+        Function that computes the maximum theoretical energy of the cell in Wh.
+
+        By integrating the OCV over the SOC, we can calculate the amount of energy available in the cell, assuming no
```

Comment:
The tone here is a bit more academic than you find in most documentation. Let's switch from active "we" form to more concise, indicative form. 

"Computes cell energy by integrating Open-Circuit Voltage (OCV) over the supplied state of charge (SOC) ranges. Assumes no energy loss, such as from the resistive or hysteresis elements."

☝🏼 Simpler because there are fewer commas. Shorter because no active voice (No us, only code).

Reply:
Changed the wording :D